extensions/wikia/Rail/RailController.class.php

```diff
@@ -8,139 +8,159 @@ class RailController extends WikiaController {
 	const FILTER_LAZY_MODULES = true;
 	const FILTER_NON_LAZY_MODULES = false;
 
-	public function executeIndex($params) {
-		wfProfileIn(__METHOD__);
+	public function executeIndex( $params ) {
+		$railModules = isset( $params['railModuleList'] ) ? $params['railModuleList'] : [];
 
-		$railModules = isset($params['railModuleList']) ? $params['railModuleList'] : [];
-
-		$this->isEditPage = isset($params['isEditPage']) ? $params['isEditPage'] : false;
-		$this->railModuleList = $this->filterModules($railModules, self::FILTER_NON_LAZY_MODULES);
+		$this->isEditPage = isset( $params['isEditPage'] ) ? $params['isEditPage'] : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)